curriculum/challenges/english/16-the-odin-project/top-the-box-model/the-box-model-question-g.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 64a6702e7a7d00f97013ed0e
+title: The Box Model Question G
+challengeType: 15
+dashedName: the-box-model-question-g
+--- 
+# --description--
+
+[lines](https://cdn.statically.io/gh/TheOdinProject/curriculum/main/foundations/html_css/css-foundations/the-box-model/imgs/odin-lined.png)
```

Comment:
```suggestion
![lines](https://cdn.statically.io/gh/TheOdinProject/curriculum/main/foundations/html_css/css-foundations/the-box-model/imgs/odin-lined.png)
```

These are showing up as links, did we want them to be images? Also, when I change this one to an image, it overflows it's container and looks a little goofy. So maybe we wanted them to be links.